### ISSUE TITLE
How to add empty labels (negative images) to YOLO-NAS training 

### ISSUE THREAD

**Issue by qwoprocks** [Deci-AI/super-gradients#1091]
### 💡 Your Question

I tried adding empty .txt files to the `labels_dir` passed in to `coco_detection_yolo_format_train`, and although the annotations are being cached, I don't see them being included in training (dataset size is less than the number of labels/images I have). 

I also tried putting a newline into each empty .txt file, and I got the EmptyDataset exception: `Empty Dataset: Out of 79035 images, not a single one was found with any of these classes: None`

Is there something else I need to do?

**Comment by BloodAxe** [comment]
You can try settings `ignore_empty_annotations: False` in your dataset params for training & validation datasety params to include empty samples.

https://github.com/Deci-AI/super-gradients/blob/4c2f8c31ac696664efd3809a7896c3f1472217ad/src/super_gradients/training/datasets/detection_datasets/detection_dataset.py#L83


```
train_dataset_params:
  data_dir: /data/coco # root path to coco data
  ...
  ignore_empty_annotations: False <-------------------- THIS IS NEW
  ...
  input_dim: [640, 640]
  ...
 
```

**Comment by qwoprocks** [comment]
That did the trick, thanks!

**Comment by BartvanMarrewijk** [comment]
This option seems to be missing for the coco_segmentation dataset. Is there a way to use negative images there as well?
